Automattic/wp-calypso
# Focused Launch: Remove temporary calypso route

#### Changes proposed in this Pull Request
We introduced feature flags to enable immediate rendering of the focused launch modal but have to be removed again before the release since it doesn't add value to production. In this PR the feature flags are removed as well as the routing configuration.

#### Technical changes
* removed `create/focused-launch-flow-calypso` flags from `config/development.json`.
* reverted changes in `client/gutenberg/editor/calypsoify-iframe.tsx` that trigger the Focused Launch Flow Modal with feature flags.
* removed `client/blocks/editor-launch-modal/index.tsx`.

#### Testing instructions

1. Check out this branch and run `yarn dev --sync` from `/apps/editing-toolkit`, ensuring you're also logged into your sandbox environment.
2. Create a new site with `/start`.
3. Sandbox your newly created site in your host file.
4. Flush your cache & DNS and then refresh your site made in step 2 and ensure that you're using the `dev` version of the Editing Toolkit.
5. Navigate to `/page/[UNLAUNCHED_SITE_ID].wordpress.com/home?flags=create/focused-launch-flow-calypso`
6. Ensure that:
    * [ ] You're not redirected to the Focused Launch Flow.
    * [ ] You're being redirected to `/start/launch-site/domains-launch?siteSlug=[UNLAUNCHED_SITE_ID].wordpress.com`
7. Create a new site with `/new`.
8. Repeat steps 3-5.
9. Ensure that:
    * [ ] You're not redirected to the Focused Launch Flow.
    * [ ] You're being redirected to the Step-By-Step Flow.

Fixes #46843
